src/locales/hi/common.json

```diff
@@ -29,14 +29,19 @@
       }
     }
   },
-  
-  "footer":{
-    "contact":"संपर्क संबंधी",
-    "terms":"नियम और शर्तें",
-    "shipping":"नौवहन नीति",
+
+  "footer": {
+    "contact": "संपर्क संबंधी",
```

Comment:
I don't think this is correct translation. What's your native language? @RenuSuresh

Reply:
yes made changes here in [pr](https://github.com/pesto-students/little-tags-RenuSuresh-front-end/pull/13/files)
![image](https://user-images.githubusercontent.com/34194473/110129707-3df3ac00-7dee-11eb-9405-768b9158698f.png)